calculator.rb

```diff
@@ -0,0 +1,142 @@
+def run_calculator
+  puts "Welcome to the calculator program!"
+  operation_select
+  run_again
+end
+
+def run_again
+  puts "\nWould you like to run the calculator again? (yes or no)"
+  input = gets.chomp
+    if input == "yes"
```

Comment:
Usually folks don't indent for the opening of an `if` block, it can make it hard to see what's currently in scope.